templates/patterns/button/button.ui_patterns.yml

```diff
@@ -23,30 +23,35 @@ button:
     splash_page:
       label: "Splash page"
       description: "Special buttons used on the site's splash page."
+    ghost:
+      label: "Ghost"
+      description: "Special buttons used on the site's splash page."
   fields:
+    label:
+      type: "text"
+      label: "Label"
+      description: "Label of the button."
+      preview: "Button"
     type_attribute:
       type: "text"
       label: "Type attribute"
       description: "The buttons type attribute"
       preview: "submit"
-    href:
-      type: "Url"
-      label: "URL"
-      description: "URL to use if button needs to be a link (A Drupal Url object)"
-      preview: "http://www.example.com"
     icon:
-      type: "text"
+      type: "array"
       label: "Icon"
-      description: "An icon to use with the button."
-      preview: "caret-right"
-    label:
+      description: "An icon."
+      preview:
+        - path: ''
+        - type: 'ui'
+        - name: 'corner-arrow'
+    icon_position:
       type: "text"
-      label: "Label"
-      description: "Label of the button."
-      preview: "Button"
-    is_block:
```

Comment:
Same here, we cannot remove the `is_block` property here, we need to make the `is_block` behave correctly using the ECL 2, so within the Twig template, as we will do with the icon above.